packages/cel/src/func.ts

```diff
@@ -98,8 +103,15 @@ export function celOverload<
   parameters: P,
   result: R,
   impl: (...args: CelValueTuple<P>) => CelInput<R>,
+  flags: CelOverloadFlag[] = [],
 ): CelOverload<P, R> {
-  return new FuncOverload(parameters, result, impl);
+  return new FuncOverload(
+    crypto.randomUUID(),
```

Comment:
It seems like `crypto` is safe to use everywhere?

Reply:
"Widely available" according to mdn. Safe to use. https://developer.mozilla.org/en-US/docs/Web/API/Crypto/randomUUID